src/Rendering/Bs4FormRenderer.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * This file is part of the Nextras community extensions of Nette Framework
+ *
+ * @license    New BSD License
+ * @link       https://github.com/nextras/forms
+ */
+
+namespace Nextras\Forms\Rendering;
+
+use Nette;
+use Nette\Forms\Controls;
+use Nette\Forms\Form;
+use Nette\Forms\Rendering\DefaultFormRenderer;
+
+/**
+ * FormRenderer for Bootstrap 4 framework.
+ */
+class Bs4FormRenderer extends DefaultFormRenderer
+{
+	/** @var Controls\Button */
+	public $primaryButton = null;
+
+	/** @var bool */
+	private $controlsInit = false;
+
+	/** @var int */
+	private $layout = null;
+
+	public function __construct($layout=FormLayout::HORIZONTAL)
```

Comment:
spaces

Reply:
Hmm, we should probably incorporate https://github.com/nette/coding-standard into Travis.